posts/2021-07-03_orkney/content.html

```diff
@@ -0,0 +1,54 @@
+<p><strong>A</strong></p> 
+<p>The Orkney islands are an archipelago consisting of 70 islands (20 inhabited) to the North East of Scotland, located between Scotland mainland and the Shetland Islands. The soil is rich, and agriculture is everywhere, and has been since ancient times. Nowadays, the Orkneys showcase neolithic sites to visit such as farmhouses of about 5000 year old, standing stones, and burying chambers. Coastal nature on the Orkneys is also amazing, there are cliffs full of birds (in the spring when they are nesting), amazing beaches and very strong tides up to about 8 knots that spice up the waters around Orkney. Special underwater generators have been installed to tab electricity off the strong tidal currents. In short, lots to see and explore. We took 2 kayaks with us, and also our tandem bike.</p>
```

Comment:
```suggestion
<p><strong>T</strong>he Orkney islands are an archipelago consisting of 70 islands (20 inhabited) to the North East of Scotland, located between Scotland mainland and the Shetland Islands. The soil is rich, and agriculture is everywhere, and has been since ancient times. Nowadays, the Orkneys showcase neolithic sites to visit such as farmhouses of about 5000 year old, standing stones, and burying chambers. Coastal nature on the Orkneys is also amazing, there are cliffs full of birds (in the spring when they are nesting), amazing beaches and very strong tides up to about 8 knots that spice up the waters around Orkney. Special underwater generators have been installed to tab electricity off the strong tidal currents. In short, lots to see and explore. We took 2 kayaks with us, and also our tandem bike.</p>
```